tests/baselines/reference/docker/azure-sdk.log

```diff
@@ -147,13 +144,22 @@ npm ERR! Failed at the @azure/keyvault-certificates@X.X.X extract-api script.
 npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
 npm ERR! A complete log of this run can be found in:
 npm ERR!     /root/.npm/_logs/XXXX-XX-XXXXXXXXX-debug.log
+@azure/service-bus (? seconds)
+>>> @azure/service-bus
+tsc -p . && rollup -c 2>&1 && npm run extract-api
+src/serviceBusClient.ts(90,34): error TS1109: Expression expected.
```

Comment:
this looks bad and is not in master

Reply:
file hasn't changed since November

Reply:
Never mind, I was looking at an old commit. There was a commit 4 days ago that added a couple of `?.` uses. This fails to parse, which makes me think that the TS we're using isn't actually latest. 

@weswigham 

1. can you take a look?
2. do you want to merge this in the meantime?

Reply:
It is now in master, so might as well fix it there.